pipeline-modules/app-service/gcp/docker-compose-build-n-push.yaml

```diff
@@ -0,0 +1,118 @@
+provisioner: gcp
+name: docker-compose-build-n-push
+version: 1
+revision: 1
+displayName: Docker Compose Build & Push docker images
+description: This plugin only use to Run docker compose  build  & push command
```

Comment:
This should be more descriptive - "Use docker-compose to build and push a container image"

Reply:
I have updated that